system/Session/Session.php

```diff
@@ -56,86 +64,151 @@ class Session implements SessionInterface
+	 * your session table's PRIMARY KEY when changing this setting.
+	 * 
+	 * @var bool 
+	 */
 	protected $sessionMatchIP = false;
 
+	/**
+	 * How many seconds between CI regenerating the session ID.
+	 * @var int 
+	 */
 	protected $sessionTimeToUpdate = 300;
 
+	/**
+	 * Whether to destroy session data associated with the old session ID
+	 * when auto-regenerating the session ID. When set to FALSE, the data
+	 * will be later deleted by the garbage collector.
+	 * @var bool 
+	 */
 	protected $sessionRegenerateDestroy = false;
 
-	protected $cookiePrefix = '';
-
+	/**
+	 * The domain name to use for cookies.
+	 * Set to .your-domain.com for site-wide cookies.
+	 * @var string
+	 */
 	protected $cookieDomain = '';
 
+	/**
+	 * Path used for storing cookies.
+	 * Typically will be a forward slash.
+	 * @var string 
+	 */
 	protected $cookiePath = '/';
 
+	/**
+	 * Cookie will only be set if a secure HTTPS connection exists.
+	 * @var bool 
+	 */
 	protected $cookieSecure = false;
 
 	/**
+	 * Logger instance to record error messages and awarnings.
 	 * @var \PSR\Log\LoggerInterface
 	 */
 	protected $logger;
 
 	//--------------------------------------------------------------------
 
-	public function __construct(\SessionHandlerInterface $driver, $config)
+	/**
+	 * Constructor.
+	 * 
+	 * Extract configuration settings and save them here.
+	 * 
+	 * @param \SessionHandlerInterface $driver
+	 * @param \Config\App $config
+	 */
+	public function __construct(\SessionHandlerInterface $driver, \Config\App $config)
 	{
 		$this->driver = $driver;
 
-		$this->sessionDriverName        = $config->sessionDriver;
-		$this->sessionCookieName        = $config->sessionCookieName;
-		$this->sessionExpiration        = $config->sessionExpiration;
-		$this->sessionSavePath          = $config->sessionSavePath;
-		$this->sessionMatchIP           = $config->sessionMatchIP;
-		$this->sessionTimeToUpdate      = $config->sessionTimeToUpdate;
+		$this->sessionDriverName = $config->sessionDriver;
+		$this->sessionCookieName = $config->sessionCookieName;
+		$this->sessionExpiration = $config->sessionExpiration;
+		$this->sessionSavePath = $config->sessionSavePath;
+		$this->sessionMatchIP = $config->sessionMatchIP;
+		$this->sessionTimeToUpdate = $config->sessionTimeToUpdate;
 		$this->sessionRegenerateDestroy = $config->sessionRegenerateDestroy;
 
-		$this->cookiePrefix = $config->cookiePrefix;
 		$this->cookieDomain = $config->cookieDomain;
-		$this->cookiePath   = $config->cookiePath;
+		$this->cookiePath = $config->cookiePath;
 		$this->cookieSecure = $config->cookieSecure;
 	}
 
 	//--------------------------------------------------------------------
 
+	/**
+	 * Initialize the session container.
+	 */
 	public function initialize()
 	{
 		if (is_cli())
 		{
 			$this->logger->debug('Session: Initialization under CLI aborted.');
 
 			return;
-		}
-		else if ((bool)ini_get('session.auto_start'))
+		} else if ((bool) ini_get('session.auto_start'))
```

Comment:
According to the styleguide the `}` should be the only character on the line, moving `else if...` down to the next line.